backbone.js

```diff
@@ -1261,8 +1261,12 @@
 
     // Produces a DOM element to be assigned to your view. Exposed for
     // subclasses using an alternative DOM manipulation API.
-    _createElement: function(tagName) {
-      return document.createElement(tagName);
+    _createElement: function(tagName, namespace) {
+        if (typeof namespace === 'undefined') {
```

Comment:
The `== null` or ternary would be better than a `typeof` check. If you check the source, Backbone uses `== null` several times.